builds/azure-pipelines.yml

```diff
@@ -62,7 +62,7 @@ steps:
 - task: UseDotNet@2
   inputs:
     packageType: sdk
-    version: 6.0.400
+    version: 7.0.0
```

Comment:
@hez2010 do you know which version we need?

Reply:
Oops I forgot to update. We can simply use `7.0.*` and set `include-prerelease: true`.

Reply:
# Resolved.

Reply:
Still failing :thinking:
We might need to specify the exact version number

Reply:
The exact sdk version number is `7.0.100-rc.2.22477.23`.

Reply:
Why MSBuild cannot find the installed sdk?

Reply:
Now things go right. 
We need to write a `UsePreviews=True` to sdk.txt in VS user data directory to enable preview SDK adoption for VS.